From b94629ed97cfaef8a9f22d18674b3d51a0d3616f Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Sep 2016 17:00:36 +0500
Subject: [PATCH] vktrace: disable Qt5 for now

We do not enable the Qt5 integration for now as
it is not needed and has build issues in the current
configuration.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 vktrace/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vktrace/CMakeLists.txt b/vktrace/CMakeLists.txt
index 21f06f7..6534d27 100644
--- a/vktrace/CMakeLists.txt
+++ b/vktrace/CMakeLists.txt
@@ -25,7 +25,7 @@ if (NOT PYTHONINTERP_FOUND)
     message(FATAL_ERROR "Missing PythonInterp. Install python interpreter 2.7 (on linux use cmd: sudo apt-get install python2.7)")
 endif()
 
-find_package(Qt5 COMPONENTS Widgets Gui Core Svg)
+#find_package(Qt5 COMPONENTS Widgets Gui Core Svg)
 if (NOT Qt5_FOUND)
     message(WARNING "Qt5 dev libraries not found, vktraceviewer will not be built.\nTo enable build of vktraceviewer on Windows, set env var Qt5_Dir to\nC:\\Qt\\5.3\\msvc2013_64\\lib\\cmake\\Qt5 or C:\\Qt\\5.3\\msvc2013\\lib\\cmake\\Qt5")
 endif()
-- 
1.9.1

